pkg/kube/configmap/configmap_builder.go

```diff
@@ -42,6 +42,13 @@ func (b *builder) SetLabels(labels map[string]string) *builder {
 	return b
 }
 
+func (b *builder) SetData(data map[string]string) *builder {
+	for k, v := range data {
```

Comment:
[nit] I think this function `SetData` could delegate to `SetDataField` for each field in the argument, instead of doing the assignment in here.

Reply:
makes sense, done